### PR TITLE
Fix error with annotateTag call and runBuild

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

File: vars/annotateTag.groovy [hunk 1]
```diff
@@ -14,7 +14,7 @@ def call(Object ctx, String imageStream, tag, name, value) {
                tags[i].annotations = [:]
            }
            tags[i].annotations[name] = value
-           echo "Annotating ImageStreamTag ${is.name()}:${tags[i].name()} with ${name}=${value}"
+           ctx.echo "Annotating ImageStreamTag ${is.name()}:${tags[i].name} with ${name}=${value}"
```
Comment (stevekuznetsov) [inline vars/annotateTag.groovy:17]: It is not clear to me when you want echo and when you want this

**Review by stevekuznetsov — COMMENTED** [review]
/lgtm

File: vars/annotateTag.groovy (hunk 1 above)
Comment (csrwng) [reply vars/annotateTag.groovy:17]: as a matter of good hygiene, if outside of the Jenkinsfile, I use the Jenkinsfile context

Comment (csrwng) [reply vars/annotateTag.groovy:17]: that wasn't the issue though... the problem is that it was throwing an error because tags[i].name is not a function.

**Pull request merged**